x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines_logic.ts

```diff
@@ -213,6 +213,7 @@ export const PipelinesLogic = kea<MakeLogicType<PipelinesValues, PipelinesAction
     },
     createMlInferencePipelineSuccess: () => {
       actions.fetchMlInferenceProcessors({ indexName: values.index.name });
+      actions.fetchCustomPipeline({ indexName: values.index.name });
```

Comment:
These are needed to ensure the JSON updated when we add / remove things from the ml inference pipeline.

Reply:
non-blocking: you can want to add a small comment explaining these are to keep data up to date.